astaire/Dockerfile

```diff
@@ -2,8 +2,10 @@ FROM clearwater/base
 MAINTAINER maintainers@projectclearwater.org
 
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes clearwater-memcached
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --force-yes astaire
 
 COPY memcached.supervisord.conf /etc/supervisor/conf.d/memcached.conf
+COPY astaire.supervisord.conf /etc/supervisor/conf.d/astaire.conf
 COPY clearwater-group.supervisord.conf /etc/supervisor/conf.d/clearwater-group.conf
 
 EXPOSE 11211
```

Comment:
I'm surprised we don't need to `EXPOSE 11311` here too, since astaire listens on 11311. Maybe this is related to the error you were seeing with multiple astaire processes being unable to talk to each other? If it's easy to test, it might be worth doing.